scylla/tests/integration/utils.rs

```diff
@@ -450,7 +450,13 @@ pub(crate) async fn execute_unprepared_statement_everywhere(
 
 /// Checks that the session is connected to a cluster with expected number of nodes,
 /// that all nodes are connected (UP) and that a simple query can be executed everywhere.
-#[cfg(all(feature = "openssl-010", feature = "rustls-023"))]
+#[cfg_attr(
+    not(any(
+        all(scylla_unstable, feature = "unstable-host-listener"),
+        all(feature = "openssl-010", feature = "rustls-023")
+    )),
+    expect(dead_code)
+)]
```

Comment:
When I see such declarations, I wonder if we can just `allow(dead_code)` for whole `ccm` module, and utils like this one. Probably not a good idea, idk.

Reply:
The problem of such declarations will disappear once we start using CCM for testing stable core features of the driver. Let's do it :)